CryceTruly/si-react
# 165118711 Users should be able to view their profiles with their parcels.

#### What does this PR do?
Adds a feature that enables users to view their profiles and their parcel order statuses
#### Description of Task to be completed?
- add profile page
- display user profile information
- display users orders

#### How should this be manually tested?
```
 Clone the repo, cd into si-react and check out the branch ft-profiles-165118711
 npm i
 npm start
 login and create an order
 go to the homepage

```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#165118711](https://www.pivotaltracker.com/story/show/165118711

#### Screenshots (if appropriate) 
<img width="1416" alt="Screenshot 2019-04-07 at 21 17 43" src="https://user-images.githubusercontent.com/20795487/55687881-b17aad80-597a-11e9-9585-a6577b477354.png">
